Mic-Python-CodeExercises/exer1.py

```diff
@@ -0,0 +1,15 @@
+import time
```

Comment:
Next time , Do the work in the actual  Problem1.py and Problem2.py files. 

Git  can track the changes of all the files in the folder you are using, dont worry. That is the power of git.

Reply:
rEVIEWING

Reply:
Type here if you see the changes

Reply:
Presss the "resolve conversation" if you have read this part..

Reply:
ahh okii kanina pa ako naliligaw kung san ba hahanapin yung comment mo xD